google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemTest.java

```diff
@@ -266,11 +265,15 @@ public void testDeleteEmptyFolder() throws IOException {
 
   @Test
   public void testDeleteFullFolder() throws IOException {
-    thrown.expect(CloudStoragePseudoDirectoryException.class);
     try (FileSystem fs = CloudStorageFileSystem.forBucket("bucket")) {
       Files.write(fs.getPath("dir/angel"), ALONE.getBytes(UTF_8));
       // we cannot delete existing folders if they contain something
       Files.delete(fs.getPath("dir/"));
+      Assert.fail();
+    } catch (CloudStoragePseudoDirectoryException ex) {
+      String message = "Can't perform I/O on pseudo-directories (trailing slash): dir/";
+      assertNotNull(ex.getMessage());
+      assertEquals(message, ex.getMessage());
```

Comment:
`assertEquals("Can't perform I/O on pseudo-directories (trailing slash): dir/", ex.getMessage());`

is enough, assertNotNull is not needed.